packages/components-react/src/icon/Icon.tsx

```diff
@@ -0,0 +1,26 @@
+import IconSet, { type IconId } from '@lux-design-system/icons';
+// import * as IconsTabler from '@tabler/icons-react';
+import { Icon as UtrechtIcon, type IconProps as UtrechtIconProps } from '@utrecht/component-library-react';
+import { PropsWithChildren, ReactNode } from 'react';
+import FallbackIcon from './FallbackIcon';
+
+export type IconProps = UtrechtIconProps & {
+  iconId: IconId | string;
+  iconLibrary?: Partial<Record<string, ReactNode>>;
+};
+
+export const Icon = ({ iconId, iconLibrary, children }: PropsWithChildren<IconProps>): ReactNode => {
```

Comment:
Waar is `children` goed voor? Wat voor content kun je in een icon plaatsen?

Reply:
Van de documentatie van Utrecht begrijp ik dat je zelf een SVG in een `Icon` component moet stoppen. Ik zou verwachten dat dat met `iconId` opgelost zou worden.

Reply:
Ik laat het er nog even in, het klopt dat het `iconId` het oplost. Je kan nu wel een combo maken van meerdere iconen bijvoorbeeld. Het RHC-component heeft het ook, daar heb ik hem op gebaseerd.